addons/calendar/static/src/js/calendar_renderer.js

```diff
@@ -110,6 +111,22 @@ const AttendeeCalendarRenderer = CalendarRenderer.extend({
         CalendarPopover: AttendeeCalendarPopover,
         eventTemplate: 'Calendar.calendar-box',
     }),
+
+    events: _.extend({}, CalendarRenderer.prototype.events, {
+        'click #google_sync_activate': '_onGoogleConfigCalendar',
+        'click #microsoft_sync_activate': '_onMicrosoftConfigCalendar',
```

Comment:
"_onConfigureExternalCalendarGoogle" and ""_onConfigureExternalCalendarMicrosoft" ?